docs/degradation_example.ipynb

```diff
@@ -1,630 +0,0 @@
-{
```

Comment:
I see that this file was modified and renamed as `docs/degradation_and_soiling_example.ipynb`. Were there any changes to the sections other than the soiling model addition (new Section 5)? 

There is an opportunity to use the soiling model as part of data normalization. This could entail correcting performance index for soiling losses as an "expected clean" performance index. For this release that idea is out of scope but is it on your roadmap?

Reply:
The addition of soiling was the main change. Another substantial change was moving plotting stuff into a plotting module.  I made this diff to show all the changes if anyone interested: https://app.box.com/s/czynwmqimduibeduavgw75w4p5fe30xc

I agree that it is interesting to consider using soiling as part of normalization. I attempted this in https://ieeexplore.ieee.org/document/8981183 but the algorithm didn't generalize well to other cases and isn't solved in a self-consistent closed loop.

@asmunds came up with a very interesting and fundamentally different approach described here: https://ieeexplore.ieee.org/document/9186286 we'd like to get it into RdTools as well (#150)